server/protocols/protocols-library/pom.xml

```diff
@@ -52,6 +52,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-webadmin-core</artifactId>
+        </dependency>
```

Comment:
I think this dependency is a bad idea: now I can no longer use IMAP without pulling webadmin.

I would prefer that we have a dedicated webadmin maven module for protocols.